rollups/evm/verify-base-contracts/setup.sh

```
@@ -0,0 +1,56 @@
+
+rm -rf minievm contracts foundry.toml
+
+# download contracts from https://github.com/initia-labs/minievm
+git clone https://github.com/initia-labs/minievm
+
+# get current minievm version from calling rpc endpoint /abci_info via HTTPS GET
+MINIEVM_VERSION=$(curl -s $RPC_ENDPOINT/abci_info | jq -r '.result.response.version')
+echo "Using MINIEVM_VERSION: $MINIEVM_VERSION from $RPC_ENDPOINT"
+
+# checkout the specific version of minievm or use main if checkout fails
+cd minievm
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)